plugins/outputs/cratedb/cratedb.go

```diff
@@ -171,12 +190,15 @@ func escapeObject(m map[string]interface{}) (string, error) {
 	// Now we build our key = val pairs
 	pairs := make([]string, 0, len(m))
 	for _, k := range keys {
-		// escape the value of our key k (potentially recursive)
-		val, err := escapeValue(m[k])
+		key := strings.ReplaceAll(escapeString(k, `"`), ".", keyReplacement)
```

Comment:
Should we maybe _first_ replace and _then_ escape the resulting string? Otherwise it might be possible to craft the replacement to circumvent escaping?

Reply:
Oh, good catch